google/oauth2/challenges.py

```diff
@@ -173,6 +192,51 @@ def obtain_challenge_input(self, metadata):
                 sys.stderr.write("No security key found.\n")
             return None
 
+    def _obtain_challenge_input_webauthn(self, metadata, webauthn_handler):
+        sk = metadata["securityKey"]
+        challenges = sk["challenges"]
```

Comment:
Are these fields always available in the JSON?  I want to make sure it has been validated previously so we don't throw a `KeyError`